src/main/java/com/google/sps/servlets/SentimentServlet.java

```diff
@@ -52,40 +63,63 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     } catch (GoogleJsonResponseException e) { // some other problem with requesting comment data
       response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, COMMENTS_FAILED_ERROR);
     }
-    String comments = String.join(". ", commentsList);
 
     String captions = captionService.getCaptionFromId(videoId);
 
-    if (comments.isEmpty() && captions.isEmpty()) {
-      VideoAnalysis videoAnalysis = new VideoAnalysis.Builder()
-          .setScore(null)
-          .setScoreAvailable(false)
-          .build();
-      String json = new Gson().toJson(videoAnalysis);
-
+    if (commentsList.isEmpty() && captions.isEmpty()) {
+      String json = createResponseJson(null, false);
       response.setContentType("application/json;");
       response.getWriter().println(json);
       return;
     }
+    
+    // Creates 26 thread pools (25 for comments, 1 for captions)
+    ExecutorService executor = Executors.newFixedThreadPool(26);
```

Comment:
Make `26` into a static constant int in the class and have the comment over there.